tutorials/dataframe/df007_snapshot.py

```diff
@@ -11,6 +11,7 @@
 
 # A simple helper function to fill a test tree: this makes the example stand-alone.
 def fill_tree(treeName, fileName):
+    df = ROOT.RDataFrame(10000)
```

Comment:
The code is not valid without that line, how did the tutorial run before?

Reply:
It didn't run! That's why I'm making this PR!!